plugins/lando-engine/test/unit/compose.spec.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * Tests for lando-engine:compose.
+ * @file compose.spec.js
+ */
+
+'use strict';
+
+// Setup chai.
+const chai = require('chai');
+const expect = chai.expect;
+chai.should();
+
+// Get the compose module to test.
+const compose = require('./../../lib/compose');
+
+const myOpts = {
```

Comment:
It might be good to add a comment here describing what these options are, even just a link to a reference